handler.go

```diff
@@ -180,6 +180,10 @@ func makeServerStream(ctx context.Context, stream *yarpctransport.ClientStream,
 		return err
 	}
 
+	if err = closeSendStream(ctx, stream, opts.DelayCloseSendStream.Duration()); err != nil {
```

Comment:
nit: `if err :=`

Reply:
if the request is unary, dos a close-send delay make sense?

Reply:
Dropped close-send delay, initially thought users could leverage it to notice request behavior through interceptors/middleware before reaching the handler.